### DISCUSSION TITLE
Getting the 404 not found error code.

### DISCUSSION THREAD

**Discussion by thickthumb** [vercel/community#4635] [Help]
### Summary

after successfully launching a project, i made an edit and pushed to git hub, redeployed the project on Vercel and been getting the 404 error code every since.. the code is 404 note found 

id: cdg1::487th-1698362628647-4592081645c8

please advice on what to do

### Steps to Reproduce

deploy a project containing 2 HTML docs connected to each other..index.html and form.html

view both to see if they link 

make an edit on the latter and push via Github

view your project again and see if it still connects.

**Comment by amyegan** [comment]
Hi @thickthumb! Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use to help with debugging?

**Reply by thickthumb** [reply]
Hi, @amyegan  fixed thank you !
